suite/tests/client-interface/attach_test.dll.c

```diff
@@ -32,7 +32,7 @@
 
 #include "dr_api.h"
 
-static client_id_t client_id = 0;
+static client_id_t client_id;
```

Comment:
I meant this variable is not used so it can be deleted entirely.

Reply:
Done